OasisDEX/oasis-borrow
# Aave params to context mapping

# [Aave params to context mapping](https://app.shortcut.com/oazo-apps/story/6704/aave-params-to-automation-mappings)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- created `AaveAutomationContext`, `MakerAutomationContext`, refactored `AutomationContextInput`
- removed protocol dependencies from `AutomationContextProvider`
- mapped Aave manage vault state to automation contex
  
## How to test 🧪
  <Please explain how to test your changes>

- current automation (Maker) should work as usuall
- automation context should be available in manage aave vault view (use `useAutomationContext` to get access to it on aave components tree)
